sdk/servicebus/service-bus/src/models.ts

```diff
@@ -245,10 +252,17 @@ export interface ServiceBusSessionReceiverOptions extends OperationOptionsBase {
    */
   receiveMode?: "peekLock" | "receiveAndDelete";
   /**
-   * The maximum duration in milliseconds
-   * until which, the lock on the session will be renewed automatically by the sdk.
+   * The maximum duration in milliseconds until which, the lock on the session will be renewed automatically by the sdk.
+   * 
    * - **Default**: `300000` milliseconds (5 minutes).
    * - **To disable autolock renewal**, set this to `0`.
+   * 
+   * **Example:**
+   *    
+   *    If the lock expires in 2 minutes and your processing time is say, 8 minutes...
+   * 
+   *    Set maxAutoLockRenewalDurationInMs to 8 minutes, and the lock will be automatically renewed for about 3 times 
```

Comment:
```suggestion
   *    Set maxAutoLockRenewalDurationInMs to 10 minutes, and the lock will be automatically renewed about 4 times 
```

We probably want to encourage taking a longer renewal duration than processing is expected to take, since timings aren't exact.